extensions/blocks/eventbrite/edit.js

```diff
@@ -273,7 +273,9 @@ class EventbriteEdit extends Component {
 							placeholder={ __( 'Enter an event URL to embed here…', 'jetpack' ) }
 							onChange={ event => this.setState( { editedUrl: event.target.value } ) }
 						/>
-						<Button isLarge type="submit">
+						// @todo Remove isDefault and isLarge when the minimum WP version supported by Jetpack
+						uses Gutenberg > 7.2
```

Comment:
Missing some `//`? 😄

Reply:
Oh, weird. I added this all on one line, but on commit it was cut into two without the extra `//`?